### ISSUE TITLE
Add option to specify mock file prefix

### ISSUE THREAD

**Issue by MEZk** [gojuno/minimock#127]
### Problem

I want to add prefix `zzz_` for each mock file and make mock files be sorted lexicographically in each directory:

```
reader.go
writer.go
reader_test.go
writer_test.gp
zzz_reader_mock_test.go
zzz_writer_mock_test.go
``` 

Currently I can't do this and I should do the following:
1. Generate mocks
2. Rename mocks `find . -type f -name "*_mock_test.go" -execdir mv {} zzz_{} ';'`

### Proposal

Add option `pr` to specify custom mock file prefix and use it [here](https://github.com/gojuno/minimock/blob/master/cmd/minimock/minimock.go#L271).

**Comment by zcolleen** [comment]
@MEZk hey! Thanks for an issue, just opened pr

**Comment by zcolleen** [comment]
Available at v3.4.3
